man/man8/zpoolprops.8

```diff
@@ -323,6 +324,53 @@ to the enabled state.
 See
 .Xr zpool-features 5
 for details on feature states.
+.It Sy features Ns = Ns Ar none | all | file Bq , Ns Ar file Ns ...
+Specifies a set of requested features for this pool. When set to
+.Sy all
```

Comment:
Once we have some feature-files in the PR, we should probably talk about what they are here, and our philosophy around what the different feature sets (compatibility sets) are.  Unless we want to do that in zpool-features.5?

Reply:
See the most recent commit for an attempt at an initial set of feature/compatiblity files (nb: they are not yet installed by the Makefiles). Broadly, I've tried to include a minimal set of "base" feature sets, with sensible aliases handled by symlinks (eg: freenas-11.3 -> freebsd-11.3, compat-2018 -> zol-0.6.5). This is based from the spreadsheet provided above by @ericloewe - Eric, do you have an opinion on what the explanatory text might look like?

Reply:
I think it's good practice to mention that compatibility files exist, but leave the long-form details to zpool-features.5, where the details of how feature flags operate are already described.

I'm thinking we can keep most of this change, but move the example file off to zpool-features.5

Reply:
I agree that having a "Compatibility feature sets" subsection in zpool-features is the best way to handle this. I've started this but am stumbling on mdoc syntax. Will hammer some more on it later.

Reply:
PTAL; moved the bulk of the discussion from `zpoolprops(8)` to `zpool-features(5)` - and reformatted from mdoc to groffman. Updated `zpool-create(8)` and `zpoolprops(8)` to reference the new section.